tests/README.md

```diff
@@ -133,6 +139,36 @@ make test-subset TESTS=hello-world,attached-connector
 make ci-tests
 ```
 
+## Running Tests from the ROOT of the Repository
+
+The root of the repository contains a main Makefile that abstracts and simplifies the process of running tests. This Makefile acts as a wrapper, delegating specific commands to the Makefile located in the `tests` directory. By using the main Makefile, you can execute test-related commands from the root of the repository without needing to navigate into the `tests` directory manually. It ensures that all necessary dependencies and configurations are properly handled before invoking the corresponding commands in the `tests` Makefile.
+
+### Variables explained
+
+- **TEST_TYPE**: The type of test to run (e.g., `ci-tests`, `e2e-tests`, `test-subset`, `test`).
+- **TEST_OPTIONS**: Additional options to pass to the test command (e.g., `TEST=hello-world`) or `TESTS=hello-world,attached-connector`).
+- **FORCE**: Force the creation of a new virtual environment, even if one already exists.
+
+### Available Commands
+
+For example:
+
+```bash
+# Run the ci-tests subset from the root
+make tests-e2e ci-tests
+
+# Run all E2E tests from the root
+make tests-e2e e2e-tests
+
+# Run a specific test from the root
+make test-e2e TEST_TYPE=test TEST_OPTIONS="TEST=hello-world"
+
+# Run a specific test with a subset of tests from the root
+make test-e2e TEST_TYPE=test-subset TEST_OPTIONS="TESTS=hello-world,attached-connector"
```

Comment:
The reason for my first comment is that it feels too complex to run samples like this one...